src/btree/node.ts

```diff
@@ -26,6 +26,19 @@ export function getRefs(node: Node): ReadonlyArray<Hash> {
   return isInternalNode(node) ? node[NODE_ENTRIES].map(e => e[1]) : [];
 }
 
+/**
+ * Describes the changes that happened to Replicache after a
+ * [[WriteTransaction]] was committed.
+ */
+export type Diff = readonly DiffOperation[];
```

Comment:
I thought we already had a `Patch` type lying around that has this same definition.

Reply:
Nevermind.